pkg/volume/util/operationexecutor/operation_executor.go

```diff
@@ -100,11 +103,16 @@ type OperationExecutor interface {
 func NewOperationExecutor(
 	kubeClient internalclientset.Interface,
 	volumePluginMgr *volume.VolumePluginMgr) OperationExecutor {
+
+	eventBroadcaster := record.NewBroadcaster()
```

Comment:
Dependencies should be passed in. In this case, have `NewOperationExecutor()` take a new parameter for the recorder. Plumb it all the way through from `cmd/kube-controller-manager/app/controllermanager.go` for attach/detach controller and from kubelet for the kubelet volume manager.